src/libcore/num/mod.rs

```diff
@@ -1994,12 +1994,8 @@ $EndFeature, "
 ```"),
             #[stable(feature = "rust1", since = "1.0.0")]
             #[inline]
-            pub fn signum(self) -> Self {
-                match self {
-                    n if n > 0 =>  1,
-                    0          =>  0,
-                    _          => -1,
-                }
+            pub const fn signum(self) -> Self {
```

Comment:
You can just invent new feature gate names with `rustc_const_unstable`, so the following should work

```suggestion
            #[rustc_const_unstable(feature = "const_signum")]
            pub const fn signum(self) -> Self {
```

Reply:
Okay, wasn't sure if I had to define it somewhere else. On #53718 it's under the  "const_int_sign" heading so I think I'll just use that?

Reply:
sounds good to me